semanticdb/semanticdb3/semanticdb3.proto

```diff
@@ -202,6 +202,7 @@ message SymbolInformation {
   repeated string members = 8;
   repeated string overrides = 9;
   Type tpe = 11;
+  repeated string annotations = 12;
```

Comment:
I wonder if we should prepare for handling annotation parameters in the future.

```proto
message Annotation {
  message ParameterList { string parameter  = 1; }
  string symbol = 1;
  repeated ParameterList args = 2;
}
```
We already store synthetics, which are trees, as syntax.

Reply:
There is also `javaArgs`.

Reply:
Here's what I would propose towards that end:

```
message Annotation {
  Type tpe = 1;
}

message AnnotatedType {
  reserved 2;
  Type tpe = 1;
  repeated Annotation annotations = 3;
}
```

Having a dedicated structure will allow us to freely add arguments when/if we are ready to support that. I should've thought about that when adding `AnnotatedType`.

The distinction between `javaArgs` and `scalaArgs` is an extremely ugly implementation detail of Scalac. I don't think there's a need to bring it to SemanticDB.

Reply:
what about the parameter name?

Reply:
Parameter names can be obtained by analyzing `tpe` and its constructor signature(s).